index.js

```diff
@@ -0,0 +1,52 @@
+var DAYS = [ 'SUN','MON', 'TUE', 'WED','THU', 'FRI', 'SAT'];
+var MONTHS = ['JAN', 'FEB', 'MAR', 'APR', 'MAY', 'JUN', 'JUL', 'AUG', 'SEP', 'OCT', 'NOV', 'DEC'];
+
+var calendarDates = document.querySelectorAll('tbody td');
+var currentDay = document.querySelectorAll('.current-day p');
+
+var d = new Date();
+var nowYear = d.getFullYear();
+var nowMonth = d.getMonth();
+var nowDate = d.getDate();
+var nowDay = d.getDay();
+var firstDay = new Date(nowYear, nowMonth, 1);
+var lastDay = new Date(nowYear, nowMonth + 1, 0);
+var markToday;
+
+function addZero(num) {
+    if(num < 10) {
+        return ('0' + num);
+    }
+    else {
+        return num;
+    }
+}
+
+function displayCalendar(){
```

Comment:
위의 함수에서는 여는 block 전에 공백을 추가하였는데, 이 곳에서는 사용하지 않으셨네요.
컨벤션에 정답은 없지만 프로젝트 내에서는 일관성을 지켜주시는게 좋습니다.
참고로 에어비엔비 컨벤션은 아래와 같이 사용하는 것을 권장합니다

```suggestion
function displayCalendar() {
```

- [https://github.com/airbnb/javascript#functions--signature-spacing](https://github.com/airbnb/javascript#functions--signature-spacing)